openshift/deployment-template.yaml

```diff
@@ -78,18 +78,23 @@ objects:
           containers:
             - name: "metrics-exporter"
               env:
-                - name: "MY_NAMESPACE"
+                - name: "THOTH_TEST_NAMESPACE"
```

Comment:
There is no `THOTH_TEST_NAMESPACE`. In the case of test environment, `THOTH_FRONTEND_NAMESPACE`, `THOTH_MIDDLETIER_NAMESPACE` and `THOTH_BACKEND_NAMESPACE` will all be set to `thoth-test-core`.